### PR TITLE
adding mass invitation script documentation

### PR DESCRIPTION
**Pull request by arthurwolf**
## Checklist:

Please tick the following before making your PR:

* [x] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [x] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

**Comment by arthurwolf** [comment]
Render:

Screenshot from 2022-01-11 21-41-29

**Comment by arthurwolf** [comment]
Render:

Screenshot from 2022-01-11 21-45-10

File: src/how-to/administrate/users.rst [hunk 1]
```diff
@@ -205,3 +205,33 @@ Then delete it:
 
    UUID=...
    curl -s -XDELETE "http://localhost:9200/directory/user/$UUID" | json_pp
+
+Mass-invite users to a team
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you need to invite members to a specific given team, you can use the ``create_team_members.sh`` Bash script, located `here <https://github.com/wireapp/wire-server/blob/develop/deploy/services-demo/create_team_members.sh>`__.
+
+This script does not create users or causes them to join a team by itself, instead, it sends invites to existing users, and when users accept the invitation, this then causes users to be added to the team as team members.
```
Comment (julialongtin) [inline src/how-to/administrate/users.rst:214]:
```suggestion
This script does not create users or causes them to join a team by itself, instead, it sends invites to potential users via email, and when users accept the invitation, they create their account, set their password, and are added to the team as team members.
```

File: src/how-to/administrate/users.rst [hunk 2]
```diff
@@ -205,3 +205,33 @@ Then delete it:
...
+
+Input is a `CSV file <https://en.wikipedia.org/wiki/Comma-separated_values>`__, in comma-separated format, in the form ``'Email,Suggested User Name'``.
+
+You also need to specify the inviting admin user, the team, the URI for the Brig (`API <https://docs.wire.com/understand/federation/api.html?highlight=brig>`__) service (Host), and finally the input (CSV) file containing the users to invite.
+
+The exact format for the parameters passed to the script is `as follows <https://github.com/wireapp/wire-server/blob/develop/deploy/services-demo/create_team_members.sh#L17>`__:
+
+* ``-a <admin uuid>``: `User ID <https://docs.wire.com/understand/federation/api.html?highlight=user%20id#qualified-identifiers-and-names>`__ in `UUID format <https://en.wikipedia.org/wiki/Universally_unique_identifier>`__ of the inviting admin. For example ``9122e5de-b4fb-40fa-99ad-1b5d7d07bae5``.
+* ``-t <team uuid>``: ID of the inviting team, same format.
+* ``-h <host>``: Base URI of brig, that is, the domain for your backend.
```
Comment (julialongtin) [inline src/how-to/administrate/users.rst:224]:
```suggestion
* ``-h <host>``: Base URI of brig's internal endpoint.
```

File: src/how-to/administrate/users.rst [hunk 3]
```diff
@@ -205,3 +205,33 @@ Then delete it:
...
+* ``-c <input file>``: file containing info on the invitees in format 'Email,UserName'.
+
+For example, one such execution of the script could look like:
+
+.. code:: sh
+
+   sh create_team_members.sh -a 9122e5de-b4fb-40fa-99ad-1b5d7d07bae5 -t 123e4567-e89b-12d3-a456-426614174000 -h http://localhost:9999 -c users_to_invite.csv 
+
```
Comment (julialongtin) [inline src/how-to/administrate/users.rst:232]:
```suggestion

Note: the 'http://localhost:9999' implies you are running the 'kubectl port-forward' given at the top of this document
.
```

**Review by julialongtin — APPROVED** [review]

**Pull request merged**